modules/library/cql/src/main/jjtree/ECQLGrammar.jjt

```diff
@@ -138,15 +162,15 @@ TOKEN:
  */
 TOKEN [IGNORE_CASE]:  /* keywords */
 {
-   < AND: "and"> |
+   < AND: "and" | "&&"> |
    < OR: "or"> |

```

Comment:
No "||" for or?

Reply:
Nope. it wasn't part of the old CQLGrammar either.
Not sure why :)